content/en/docs/reference/kubectl/overview.md

```diff
@@ -373,6 +373,16 @@ kubectl logs <pod-name>
 kubectl logs -f <pod-name>
 ```
 
+`kubectl diff` - Diff configurations with the current online configuration.
```

Comment:
Let's see how this conversation finishes: https://github.com/kubernetes/website/pull/18481/files#r364340730